frontend/views/containers/chatroom/CreatePoll.vue

```diff
@@ -273,7 +273,10 @@ export default {
   height: 0;
   pointer-events: none;
   overflow: hidden;
-  z-index: $zindex-modal;
+  // NOTE: z-index should be bigger than the one of sidebar
+  //       because CreatePool modal is rendered before the sidebar
+  // REF:  https://github.com/okTurtles/group-income/issues/1843
+  z-index: $zindex-modal + 1;
```

Comment:
Not sure if it's better to use `$zindex-sidebar + 1` but it's what we discussed in [here](https://github.com/okTurtles/group-income/issues/1843#issuecomment-1939568088).

Reply:
A minor typo in the comment here btw:

`CreatePool` -> `CreatePoll`